### PR TITLE
Added gaussian filter for tilt series

### PR DESCRIPTION
**Pull request by yijiang1**
Added a data processing function that applies a 2D Gaussian filter to each tilt image.

### CODE REVIEW

File: tomviz/python/GaussianFilterTiltSeries.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name" : "GaussianFilter",
+  "label" : "Gaussian Filter",
```
Comment (yijiang1) [inline tomviz/python/GaussianFilterTiltSeries.json]: What are the purposes of "name" and "label"? 
I don't quite understand how they control the UI.

Comment (yijiang1) [inline tomviz/python/GaussianFilterTiltSeries.json]: Could anyone explain this to me? When I edit these two fields, the UI often disappear like this
screen shot 2016-10-24 at 10 39 02 am
Thanks!

Comment (cquammen) [inline tomviz/python/GaussianFilterTiltSeries.json]: As described in https://github.com/OpenChemistry/tomviz/blob/master/tomviz/python/Operators.md, `name` is the internal name for the Operator, and `label` is the name that appears in the UI. `name` must not contain spaces.

Editing these fields should not cause the UI to disappear unless the JSON becomes ill formed. In that case, you should get an error message indicating the JSON is invalid.

Comment (yijiang1) [inline tomviz/python/GaussianFilterTiltSeries.json]: Turns out the empty UI above is caused by wrong format in Mac's TextEdit (default for .json file). For example, the double quote character in TextEdit becomes 
screen shot 2016-10-26 at 12 37 11 pm

Note to self: use Xcode to edit .json file.

Comment (cquammen) [inline tomviz/python/GaussianFilterTiltSeries.json]: Ah, good find. Thanks for tracking this down.

**Pull request merged**